### PR TITLE
Adding issue and pull request templates as well as code of conduct and license

### PR DESCRIPTION
**Pull request by gabelula**
Adds templates for issue and pull request creation as well as the code of conduct for Tor.

## Proposed Changes
  - Adds issue template that includes to standarize how issues are being submited.
  - Adds pull request template to include tickets associated as well as proposed changes.
  - Adds code of conduct to the root of the repository.
  - Adds License to the root of the repository.  Issue: https://github.com/ooni/explorer/issues/40

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Fixes issue #
```
Comment (hellais) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think the `#` should go before the string here.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Copyright 2017 Open Observatory of Network Interference (OONI), The Tor Project
```
Comment (hellais) [inline LICENSE]: The date should be bumped to 2019. I will do that in https://github.com/ooni/license too.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
+============================================================
+Tor Project Code of Conduct
+============================================================
```
Comment (hellais) [inline CODE_OF_CONDUCT.md]: We should probably reformat this to be markdown or maybe just link to the canonical version of this hosted on some torproject website.

Comment (gabelula) [reply CODE_OF_CONDUCT.md]: Changing it to markdown.

**Pull request merged**